syntax = "proto3";
package cs;

import "base.proto";

// SC_NOTIFY = 10000
message Notify
{
    ChatMessage chat_message = 1;
}